packages/data/src/redux-store/metadata/actions.js

```diff
@@ -36,9 +36,9 @@ export function finishResolution( selectorName, args ) {
  * Returns an action object used in signalling that a batch of selector resolutions has
```

Comment:
These changes were generated by `npm run lint-js:fix`. They are unrelated to the implementation details of the component being tested, or the tests.